test/e2e/node_feature_discovery.go

```diff
@@ -44,8 +44,8 @@ import (
 )
 
 var (
-	dockerRepo    = flag.String("nfd.repo", "quay.io/kubernetes_incubator/node-feature-discovery", "Docker repository to fetch image from")
-	dockerTag     = flag.String("nfd.tag", "e2e-test", "Docker tag to use")
+	dockerRepo    = flag.String("nfd.repo", "k8s.gcr.io/nfd/node-feature-discovery", "Docker repository to fetch image from")
+	dockerTag     = flag.String("nfd.tag", "v0.6.0", "Docker tag to use")
```

Comment:
Hmm, we should probably use `gcr.io/k8s-staging-nfd/node-feature-discovery:master` as the default here as a past release is not guaranteed to be compatible with tests from `master`. Another option would be to leave the both arg defaults empty

Reply:
I like master as default, the `Makefile` uses a proper flag to point it to the dirty git stage (when developing something) 
I was here really thinking in the casual users trying to understand the project.
will set it to `master`

Reply:
Mm, that's a fair point. This is just yet another place which needs to be modified when creating a release branch. I prolly need to create a helper script for this 🤔